packages/apps/graph/src/graph/query/fungible-account-by-public-key.ts

```diff
@@ -0,0 +1,95 @@
+import { prismaClient } from '@db/prisma-client';
+import { getFungibleChainAccount } from '@services/account-service';
+import { chainIds } from '@utils/chains';
+import { normalizeError } from '@utils/errors';
+import { builder } from '../builder';
+import FungibleAccount from '../objects/fungible-account';
+import type { FungibleChainAccount } from '../types/graphql-types';
+import { FungibleAccountName } from '../types/graphql-types';
+
+builder.queryField('fungibleAccountByPublicKey', (t) =>
```

Comment:
Do we want to add the same but on the chain level?

`fungibleChainAccountByPublicKey`

Reply:
Yep, in fact I think it makes sense to also align with the rest of the existing queries. I'll update it